components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.cpp

```diff
@@ -63,6 +68,10 @@ void Nudging::set_grids(const std::shared_ptr<const GridsManager> grids_manager)
   if (ekat::contains(m_fields_nudge,"U") or ekat::contains(m_fields_nudge,"V")) {
     add_field<Updated>("horiz_winds",   horiz_wind_layout,   m/s,     grid_name, ps);
   }
+
+  if (m_use_weights) 
+    add_field<Updated>("nudging_weights", scalar3d_layout_mid, A, grid_name, "weights", ps);
```

Comment:
Why is this field exposed? It seems to me that this should be a field local to the atm proc.

Reply:
yes, this can be the local field, if we have the time-independent nudging weights, so it doesn't need to be the global.